core/contracts/oracle/implementation/Store.sol

```diff
@@ -25,8 +25,8 @@ contract Store is StoreInterface, Withdrawable {
 
     enum Roles { Owner, Withdrawer }
 
-    FixedPoint.Unsigned public fixedOracleFeePerSecond; // Percentage of 1 E.g., .1 is 10% Oracle fee.
-    FixedPoint.Unsigned public weeklyDelayFee; // Percentage of 1 E.g., .1 is 10% weekly delay fee.
+    FixedPoint.Unsigned public fixedOracleFeePerSecondPerPfc; // Percentage of 1 E.g., .1 is 10% Oracle fee.
+    FixedPoint.Unsigned public weeklyDelayFeePerPfc; // Percentage of 1 E.g., .1 is 10% weekly delay fee.
```

Comment:
nit: shouldn't this variable also be changed to `weeklyDelayFeePerSecondPerPfc`?

Reply:
🤦 I've changed this so many times, sorry this got through, I have updated accordingly.